### PR TITLE
Spring 2019 Fanart Contest

### PR DESCRIPTION
**Pull request by Ephemeralis**
:eyes:

---

[The Spring 2019 Fanart Contest is now taking submissions for the next 14 days!](https://osu.ppy.sh/community/contests/78)

### CODE REVIEW

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+We mean spring-themed as in the season spring, not springs as in bouncy coils or whatever else.
+
+All entries that follow the rules laid out below will go up for 7 days of intense voting, and a selection of the highest-scoring winners will make it into the osu! game client as the Spring 2019 seasonal backgrounds. To celebrate, their authors will also receive **2 months of osu! supporter**. Nice!
+
+Want to enter? Read the rules below to get started, then [head on over to the contest listing page](https://osu.ppy.sh/community/contests/78) and submit your entry when you're ready. Oh, and by the way, **read the rules**. It's worth saying twice because we *always* have at least a dozen entries that never make it to voting for not following the rules!
+
+## Rules
+
+* **All entries MUST be OBVIOUSLY SPRING THEMED in some way.** Spring here refers to the season, not anything else. Flowers, sunshine, that sort of thing. Your work doesn't have to include flowers or sunshine specifically, but it does need to be "Spring" somehow. How you get there is entirely up to you, just make sure it's obvious!
```
Comment (cl8n) [inline news/2019-05-10-spring-2019-fanart-contest.md]:
```suggestion
* **All entries must be obviously spring themed in some way.** Spring here refers to the season, not anything else. Flowers, sunshine, that sort of thing. Your work doesn't have to include flowers or sunshine specifically, but it does need to be "spring" somehow. How you get there is entirely up to you, just make sure it's obvious!
```
it's already the first rule, the double emphasis probably isn't necessary

Comment (cl8n) [reply news/2019-05-10-spring-2019-fanart-contest.md]: ok same with the others

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* **All entries must be suitable for all-ages viewing.** Several thousand people will be looking at your entries, some of them will be children. Keep it clean and tasteful!
+* **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** These templates show where the big osu! cookie will be overlayed during the voting process and if your background makes it to the main-menu.
```
Comment (cl8n) [inline news/2019-05-10-spring-2019-fanart-contest.md]:
```suggestion
* **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.**
```
redundant

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+title: "Spring 2019 Fanart Contest"
+date: 2019-05-10 12:00:00 +0000
+---
+
+Better late than never! Spring might be on the way out, but talent is *always* in season, so whip out your drawing implement of choice and draw any of the osu! mascots enjoying Spring for fun and prizes!
+
+[![](https://assets.ppy.sh/contests/78/header.jpg)](https://osu.ppy.sh/community/contests/78)
+
+If you find yourself a person of artistic interest, this should be right up your alley! The artists among us have **14 days** to take a whirl at drawing any **spring-themed rendition of the osu! official or community mascots**.
```
Comment (nyquillerium) [inline news/2019-05-10-spring-2019-fanart-contest.md]: I would prefer the adjectives for mascots be swapped in order, like "official or community-sourced osu! mascots".

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** These templates show where the big osu! cookie will be overlayed during the voting process and if your background makes it to the main-menu.
+* **DO NOT LEAVE THE DOTTED osu! COOKIE TEMPLATE VISIBLE IN YOUR SUBMISSION!** Please don't do this. It doesn't need to be there when you're done drawing, it just makes your work look worse.
+* **ACCOUNT FOR THE DOTTED osu! COOKIE TEMPLATE IN THE COMPOSITION OF YOUR WORK!** We've already said this, but anything inside the dotted circle will NOT be seen during voting phase and will be very difficult to see on the main menu. You still need to draw something there - don't just leave it blank, but be creative about it. Check out previous contest entries for inspiration!
```
Comment (cl8n) [inline news/2019-05-10-spring-2019-fanart-contest.md]:
```suggestion
* **ACCOUNT FOR THE DOTTED osu! COOKIE TEMPLATE IN THE COMPOSITION OF YOUR WORK!** We've already said this, but anything inside the dotted circle will *not* be seen during voting phase and will be very difficult to see on the main menu. You still need to draw something there—don't just leave it blank, but be creative about it. Check out previous contest entries for inspiration!
```

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+* **Entries MUST meet the resolution requirements for a main-menu piece - namely 2732x1536.** We were really firm about this last contest and we'll continue to be firm about it - CHECK YOUR OUTPUT RESOLUTION BEFORE SUBMITTING! If it's off by even a few pixels, you risk having your work disqualified. A script handles this checking and it is a very angry script.
```
Comment (cl8n) [inline news/2019-05-10-spring-2019-fanart-contest.md]:
```suggestion
* **Entries must be 2732x1536.** We were really firm about this last contest and we'll continue to be firm about it—*check your output resolution before submitting*! If it's off by even a few pixels, you risk having your work disqualified. A script handles this checking and it is a very angry script.
```

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+* **SUBMIT ONLY YOUR OWN WORK! NO TRACING! NO REDRAWING! NO COMPOSITE WORKS!** Repurposing other people's pre-drawn works to make them look like osu! mascots is literally plagiarism. Using brushes and other helpful assets is fine, but if you're doing something like redrawing Miku from the Quintessential Quintuplets by redrawing a shot from the manga to look like Mocha and submitting it as your own work, you're a gigantic butthole and nobody likes you.
+* **Don't use copyrighted characters from other intellectual properties.** You can freely use any of the osu! mascots (pippi, Yuzu, Mocha, Maria) and any of the community mascots (Tama, Aiko, Taikonator, so on), but stay away from using direct representations of characters from other worlds or franchises in your work. Remember that the community mascots are also someone else's creation, so make sure to treat their work with the same respect that you'd treat yours.
+
+[Head on over to the Spring 2018 Fanart Contest listing and enter now!](https://osu.ppy.sh/community/contests/78)
```
Comment (cl8n) [inline news/2019-05-10-spring-2019-fanart-contest.md]:
```suggestion
[Head on over to the Spring 2019 Fanart Contest listing and enter now!](https://osu.ppy.sh/community/contests/78)
```

File: news/2019-05-10-spring-2019-fanart-contest.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+Better late than never! Spring might be on the way out, but talent is *always* in season, so whip out your drawing implement of choice and draw any of the osu! mascots enjoying Spring for fun and prizes!
+
+[![](https://assets.ppy.sh/contests/78/header.jpg)](https://osu.ppy.sh/community/contests/78)
+
+If you find yourself a person of artistic interest, this should be right up your alley! The artists among us have **14 days** to take a whirl at drawing any **spring-themed rendition of the osu! official or community mascots**.
+
+We mean spring-themed as in the season spring, not springs as in bouncy coils or whatever else.
+
+All entries that follow the rules laid out below will go up for 7 days of intense voting, and a selection of the highest-scoring winners will make it into the osu! game client as the Spring 2019 seasonal backgrounds. To celebrate, their authors will also receive **2 months of osu! supporter**. Nice!
```
Comment (Joehuu) [inline news/2019-05-10-spring-2019-fanart-contest.md]:
```suggestion
All entries that follow the rules laid out below will go up for 7 days of intense voting, and a selection of the highest-scoring winners will make it into the osu! game client as the Spring 2019 seasonal backgrounds. To celebrate, their authors will also receive **2 months of osu!supporter**. Nice!
```

File: news/2019-05-10-spring-2019-fanart-contest.md (hunk 1 above)
Comment (Ephemeralis) [reply news/2019-05-10-spring-2019-fanart-contest.md]: it's necessary, people STILL fuck it up even in caps and bold.

**Pull request merged**